### DISCUSSION TITLE
如何添加 alist ？

### DISCUSSION THREAD

**Discussion by nobodysclown** [Hiram-Wong/ZyPlayer#89] [Q&A]
请问v3.2.10 版本如何配置 alist 网盘

**Comment by Hiram-Wong** [comment]
<img width="1000" alt="image" src="https://github.com/Hiram-Wong/ZyPlayer/assets/30908279/e13dcdb3-e5c5-4848-9595-078c75f21615">

**Reply by nobodysclown** [reply]
想挂载自己搭建的小雅alist，可以看下配置吗，我这样填的读取不到。

![Snipaste_2023-10-03_09-26-06](https://github.com/Hiram-Wong/ZyPlayer/assets/15608416/1fde5549-c163-474e-8b5b-71b4a4092843)

**Reply by Hiram-Wong** [reply]
用元数据块加密，而不是账号密码

**Reply by arpir** [reply]
「用元数据块加密，而不是账号密码」啥意思啊

**Reply by maomaocris** [reply]
![zyplayer](https://github.com/user-attachments/assets/718a45e2-084e-4e94-a7d6-ad833b07b392)
路径里面两个//,导致我打不开文件？

**Comment by Asukaqwq** [comment]
![QQ截图20240101165221](https://github.com/Hiram-Wong/ZyPlayer/assets/110525673/a4f26d22-b087-427a-ae27-31fa70fbde4e)
你需要把游客的权限打开然后在元信息这添加你的路径
![image](https://github.com/Hiram-Wong/ZyPlayer/assets/110525673/6c5487c0-098a-4185-ba84-9ca226073c9d)

**Reply by Asukaqwq** [reply]
zyplayer加密是不用填的，相当于你以游客的身份访问你的alist
